src/components/FeatureStore/FeatureStore.js

```diff
@@ -191,10 +193,13 @@ const FeatureStore = ({
   )
 
   useEffect(() => {
-    fetchData({ project: match.params.projectName, tag: 'latest' })
+    fetchData({
+      tag: 'latest',
+      iter: match.params.pageTab === DATASETS_TAB && 'iter'
```

Comment:
This looks like a mistake.
`'iter'` is always truthy. Adding `&& 'iter'` has no meaning.